handbook/communication/company_meeting.md

```diff
@@ -1,8 +1,10 @@
-# Weekly team meetings
+# Company meeting
 
-Schedule: 10:30-11:00am PST/PDT Monday (link in the calendar event)
-
-If Monday is a holiday, the meeting is held on the next non-holiday day.
+1. Schedule: every Monday 10:30-11:00am PST/PDT (link in the calendar event)
+   - If Monday is a holiday, the meeting is held on the next non-holiday day.
+   - There is no company meeting held between December 25 and December 31. The first company meeting of the year is held on the first non-holiday Monday of the year.
+1. The link to the slides is posted in #general each Thursday before.
+1. We used to call this "team meeting", but now we call it "company meeting" because "team meeting" sounds like it's only for one specific team inside the company.
```

Comment:
I don't think this is necessary to include in evergreen docs for all future teamates. This change can be communicated in Slack to our current team.
```suggestion
```

Reply:
I will link to this point in the handbook when reminding people, therefore I want it to be in the handbook. We can remove it in 3-6 months.